### PR TITLE
feat: add new DropdownButton component and workspace icons

### PR DESCRIPTION
**Pull request by sylvanasantos**
### Overview

- Added new DropdownButton component
- Added new AddBoldIcon, BrowserPageLayoutIcon, and FilePyIcon

The new component and icons are going to be used for the new Workspaces page/sections. See monolith pr [here](https://github.com/codecademy-engineering/Codecademy/pull/24319).

See icons in storybook [here](https://60954e4501560262b03f549c--gamut-preview.netlify.app/storybook/?path=/docs/atoms-icons-regular--editable-icon).
See DropdownButton story [here](https://60954e4501560262b03f549c--gamut-preview.netlify.app/storybook/?path=/docs/molecules-dropdownbutton--dropdown-button).

SVGs used: 
Python Icon: https://drive.google.com/drive/u/1/folders/1CVNNDZbk58BOR5U_VFfJ7lumD9wrlWOS
Html/CSS/JSS Icon: https://drive.google.com/drive/u/1/folders/1KduhOVO0f25-8QHnSRqlQvO-lb_nE-hv
Add Icon: https://drive.google.com/drive/u/1/folders/1nVjVbBFhnMKHlJ1tjwWS_IM_D5WCZKZp

### PR Checklist

- [ ] Related to designs: [figma](https://www.figma.com/file/hPv0Q1QJZa3nfcxsmc8dHV/Build?node-id=16%3A1)
- [ ] Related to JIRA ticket: EGG-1103
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.11.1-alpha.fa63b3.0<br/>@codecademy/gamut-kit@0.3.85-alpha.fa63b3.0<br/>@codecademy/gamut-labs@15.6.2-alpha.fa63b3.0<br/>@codecademy/gamut@32.15.1-alpha.fa63b3.0<br/>@codecademy/styleguide@37.1.1-alpha.fa63b3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6094ff1b34e2e909332a8ef1--gamut-preview.netlify.app>https://6094ff1b34e2e909332a8ef1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6094ff1b34e2e909332a8ef1>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60953437e59af358b80a637e--gamut-preview.netlify.app>https://60953437e59af358b80a637e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60953437e59af358b80a637e>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.11.1-alpha.853b0f.0<br/>@codecademy/gamut-kit@0.3.85-alpha.853b0f.0<br/>@codecademy/gamut-labs@15.6.2-alpha.853b0f.0<br/>@codecademy/gamut@32.15.1-alpha.853b0f.0<br/>@codecademy/styleguide@37.1.1-alpha.853b0f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60954647a4ec1e66d6f9b3ba--gamut-preview.netlify.app>https://60954647a4ec1e66d6f9b3ba--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60954647a4ec1e66d6f9b3ba>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.11.1-alpha.b30c5b.0<br/>@codecademy/gamut-kit@0.3.85-alpha.b30c5b.0<br/>@codecademy/gamut-labs@15.6.2-alpha.b30c5b.0<br/>@codecademy/gamut@32.15.1-alpha.b30c5b.0<br/>@codecademy/styleguide@37.1.1-alpha.b30c5b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60954e4501560262b03f549c--gamut-preview.netlify.app>https://60954e4501560262b03f549c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60954e4501560262b03f549c>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://609572caa4abf99f1c02427b--gamut-preview.netlify.app>https://609572caa4abf99f1c02427b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/609572caa4abf99f1c02427b>Deploy Logs</a></p>

**Review by natalieweesh — APPROVED** [review]
so good! it might be nice to also include a story with icons in the dropdown 😺

File: packages/gamut-labs/src/DropdownButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { Box, FillButton, StrokeButton } from '@codecademy/gamut';
+import { AnimatePresence, motion } from 'framer-motion';
+import React, { useRef, useState } from 'react';
+
+import { Popover } from '../Popover';
+import { DropdownItem, DropdownList } from './DropdownList';
+
+export type DropdownButtonProps = {
+  buttonType?: 'fill' | 'stroke';
```
Comment (saghdaey) [inline packages/gamut-labs/src/DropdownButton/index.tsx:9]: Would it be possible to extend and also use for an icon button?

Comment (saghdaey) [reply packages/gamut-labs/src/DropdownButton/index.tsx:9]: (new sandbox file navigator designs^)

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://609929fae09cde747bc3d7aa--gamut-preview.netlify.app>https://609929fae09cde747bc3d7aa--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/609929fae09cde747bc3d7aa>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.11.2-alpha.14179e.0<br/>@codecademy/gamut-kit@0.3.87-alpha.14179e.0<br/>@codecademy/gamut-labs@15.7.2-alpha.14179e.0<br/>@codecademy/gamut@32.15.2-alpha.14179e.0<br/>@codecademy/styleguide@37.1.3-alpha.14179e.0<br/></pre>

**Comment by sylvanasantos** [comment]
@natalieweesh agreed, but unfortunately, there's a bug on storybook/gamut where icons seem to break stories? i actually had icons in the story and then removed them 😅 

@saghdaey i'm gonna merge without adding icon buttons as an option so that we can have a more in depth discussion and also keep this pr light, but fwiw i think it makes sense to just extend this to use them rather than making a whole new component. although we would have to give it a dark mode.

**Pull request merged**